### DISCUSSION TITLE
pages/Fault-Analyzer-for-zOS/FA-host_doc_output/accessibility

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#44] [Q&A]
# pages/Fault-Analyzer-for-zOS/FA-host_doc_output/accessibility

Accessibility features assist users who have a disability, such as restricted mobility or limited vision, to use information technology content successfully.

https://github01.hclpnp.com/pages/Fault-Analyzer-for-zOS/FA-host_doc_output/accessibility.html

**Comment by LouIentile** [comment]
This is another comment
